docs/chapter-2.mdx

```diff
@@ -70,14 +71,58 @@ Since Angular is heavily pushing towards standalone components, let's remove our
 
 ## Up and running
 
-Finally, all we have to do is to add our (empty) store to the app before launching it by calling `provideStore` from the application's providers:
+Finally, all we have to do is to add our (empty) store to the app before launching it:
+
+<Tabs
+    defaultValue="standalone"
+    values={[
+        {label: 'With standalone components', value: 'standalone'},
+        {label: 'From the AppModule', value: 'appmodule'},
+    ]}>
+<TabItem value="standalone">
+
+In your `main.ts` file, add the `provideStore` from `@ngrx/store` to the application's providers:
 
 ```ts {2} title="main.ts"
 bootstrapApplication(AppComponent, {
   providers: [ provideStore() ]
 }).catch((err) => console.error(err));
 ```
 
+Alternatively, you can use the former way of adding the store using `StoreModule` by calling it like this:
+
+```ts {2-4} title="main.ts"
+bootstrapApplication(AppComponent, {
+  providers: [
+    importProvidersFrom(StoreModule.forRoot({})),
```

Comment:
I don't think we need to provide an example on this; at least let's mention this is not something we recommend to do